.github/workflows/dapr-pubsub.yaml

```diff
@@ -0,0 +1,71 @@
+name: dapr-pubsub
+on:
+  push:
+    paths:
+      - "pkg/pubsub/dapr"
+      - "test/pubsub/**"
+  pull_request:
+    paths:
+      - "pkg/pubsub/dapr"
+      - "test/pubsub/**"
+permissions: read-all
+
+jobs:
+  dapr_test:
+    name: "Dapr pubsub test"
+    runs-on: ubuntu-22.04
+    timeout-minutes: 15
+    strategy:
+      matrix:
+        HELM_VERSION: ["3.7.2"]
+        GATEKEEPER_NAMESPACE: ["gatekeeper-system"]
+        DAPR_VERSION: ["1.10"]
+        AUDIT_CONNECTION: ["audit"]
```

Comment:
are we expecting to test multiple values for AUDIT_CONNECTION and CHANNEL?

Reply:
As of now, we are not. But we discussed the possibility of having multiple channels and connections for audit to route violation belonging to specific constraint and template to route to specific channels using the particular connection.

Reply:
i think it defaults to "audit" in makefile, so we should be just okay to remove entirely. same goes for HELM_VERSION and GATEKEEPER_NAMESPACE

Reply:
Removed it.